### ISSUE TITLE
Running faster on concrete or asphalt

### ISSUE THREAD

**Issue by KeroBetick** [DrMango14/Create-The_Factory_Must_Grow#63]
This is not a problem, just suggestions for development!
Concrete and asphalt are now just blocks that can be used for roads, decorations, etc.
So why not add a little acceleration to the player running along these blocks? Let's say 10-15% acceleration

**Comment by DrMango14** [comment]
yeah that would be cool but i dont want IE fans complaining about me copying them
